classes/ActionScheduler_FinishedAction.php

```diff
@@ -3,16 +3,19 @@
 /**
  * Class ActionScheduler_FinishedAction
  *
- * Stored action which has finished, which could be because it was cancelled, completed or failed.
+ * Deprecated to avoid ambiguity with stored actions and objects instantiated purely to call the hook.
+ *
+ * @deprecated 1.6.0
  */
-class ActionScheduler_FinishedAction extends ActionScheduler_StoredAction {
+class ActionScheduler_FinishedAction extends ActionScheduler_Action {
```

Comment:
If possible, this should be `ActionScheduler_Finished_Action` for consistency.

Reply:
@JPry I think you may be confusing Action Scheduler's consistency with WP standards and/or Subscriptions? See other comments about use of underscores in AS. 🙂